Godeps/Godeps.json

```diff
@@ -26,12 +26,12 @@
 		},
 		{
 			"ImportPath": "cloud.google.com/go/compute/metadata",
-			"Comment": "v0.1.0-115-g3b1ae453",
+			"Comment": "v0.1.0-115-g3b1ae45",
```

Comment:
something weird happened here... it seems like the abbreviated commit hash was shortened on all deps. Maybe you're using a different version of godep?

Reply:
```
$ godep version
godep v80 (linux/amd64/go1.11.1)
$ go version
go version go1.11.1 linux/amd64
```

I'm not sure where is wrong.

Reply:
I'm trying to figure out why.

Reply:
hi, @tallclair 

After retry timed out tests, all verify checks passed now.

Did some searching, I found the commit hash is 7 bytes instead of 8 bytes originally (omit `g` prefix). In [a recent PR](https://github.com/kubernetes/kubernetes/pull/68431/files), commit hash had been updated to 8 bytes. However, either 7 or 8 bytes passed verify checks. 

Going through [godep](https://github.com/tools/godep/blob/master/vcs.go) source quickly, I found `godep` use `git describe --tags <commit>` to get comment, so length of commit hash in comment depends on git not godep.

By default, originally `git describe` uses 7 hexadecimal digits as the abbreviated object name, but after [this commit](https://github.com/git/git/commit/e6c587c733b4634030b353f4024794b08bc86892), git use dynamic default abbreviation size (depends on git repository).

I think it's ok to update comment in Godeps.json file in current situation, because it does not break anything. To make sure length commit hash to be consistent, we need add `--abbrev=<fixsize>` flag in `godep`. Another pr has similar discussion: https://github.com/kubernetes/kubernetes/pull/69976#pullrequestreview-167565874.